<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>r168s</class>
 <widget class="QDialog" name="r168s">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1000</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Конфигурация Р168</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>991</width>
     <height>751</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Настройка Р168</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Порт R168</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="r168cfgport">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
               </font>
              </property>
              <property name="text">
               <string>АПД Вход</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QComboBox" name="ApdInputFL">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>ФЛ1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>фЛ2</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="ApdInputKR">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>КР1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>КР2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QComboBox" name="ApdInputTch">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>ТЧ1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>ТЧ2</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="ApdInputKR_2">
                <property name="font">
                 <font>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <item>
                 <property name="text">
                  <string>КР1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>КР2</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="font">
               <font>
                <pointsize>22</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Р168(Канал 1)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="SelectDevCh1">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string>МН825</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Apd TCH</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="text">
             <string>АПД Выход</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QComboBox" name="ApdOutputFL">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>ФЛ1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ФЛ2</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ApdOutputKR">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>КР1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>КР2</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QComboBox" name="ApdOutputTch">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>ТЧ1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ТЧ2</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="ApdOutputKR_2">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <item>
             <property name="text">
              <string>КР1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>КР2</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <pointsize>22</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Р168(Канал 2)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="SelectDevCh2">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>МН825</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Apd TCH</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
